src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$el-size: 100px;
+$el-padding: 10px;
+$el-gap: 1px;
+$week-days:
+  "mon" 0,
+  "tue" $el-size + $el-gap,
+  "wed" $el-size*2 + $el-gap*2,
+  "thu" $el-size*3 + $el-gap*3,
+  "fri" $el-size*4 + $el-gap*4,
+  "sat" $el-size*5 + $el-gap*5,
+  "sun" $el-size*6 + $el-gap*6;
```

Comment:
it's easy to make a mistake here, you should do these calculations inside a loop, just store the number for the day here,